js/src/modals/CreateAccount/store.js

```diff
@@ -220,7 +226,28 @@ export default class Store {
     this.stage--;
   }
 
-  createAccount = () => {
+  createAccount = (vaultStore) => {
```

Comment:
I prefer passing it here when we create, only used at this point and set by the caller as an option.